api_definition.go

```diff
@@ -281,17 +281,17 @@ func (a APIDefinitionLoader) FromDashboardService(endpoint, secret string) []*AP
 	}
 
 	// Process
-	var apiSpecs []*APISpec
+	var specs []*APISpec
```

Comment:
👍 I think we these change we eliminate a big confusion as it was not clear that we are "hiding" global apiSpecs by declaring local var